modules/reference/pages/feature/sessionDatabase-1.0/examples.adoc

```diff
@@ -0,0 +1,22 @@
+== Examples
+
+=== Minimum configuration to persist session data to a database
+
+The following example shows the minimum configuration for database persistence:
+
+[source, java]
+----
+<fileset id="DerbyFiles" includes="*.jar" dir="server1/derby/client"/>
+<library id="DerbyLib" filesetRef="DerbyFiles"/>
+<jdbcDriver id="DerbyDriver" libraryRef="DerbyLib"/>
+<dataSource id="SessionDS" jdbcDriverRef="DerbyDriver">
+    <properties.derby.client user="user1" password="password1"
+                             databaseName="server1/databases/SessionDB"
```

Comment:
here recommend

`databaseName="SessionDB"`

Reply:
Thanks. I made the updates.